# Information specific to each joint.
# This is designed to forward information from each joint to the browser.
# Examples include position, velocity, and effort.
#
# j#: the value to be published from joint #
float32 j0
float32 j1
float32 j2
float32 j3
float32 j4
float32 j5
float32 j6